frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -161,11 +161,23 @@ describe("collection permissions", () => {
 
               describe("archive", () => {
                 it("should be able to archive/unarchive question (metabase#15253)", () => {
-                  archiveUnarchive("Orders");
+                  archiveUnarchive("Orders", "question");
                 });
 
                 it("should be able to archive/unarchive dashboard", () => {
-                  archiveUnarchive("Orders in a dashboard");
+                  archiveUnarchive("Orders in a dashboard", "dashboard");
+                });
+
+                it("should be able to archive/unarchive model", () => {
+                  cy.skipOn(user === "nodata");
+                  cy.createNativeQuestion({
+                    name: "Model",
+                    dataset: true,
+                    native: {
+                      query: "SELECT * FROM ORDERS",
+                    },
+                  });
+                  archiveUnarchive("Model", "model");
```

Comment:
@nemanjaglumac can you please help me with that? The test should actually work for the "nodata" user, but `createNativeQuestion` would fail because "nodata" has no data permission 🤷‍♂️ We do the `cy.signIn` pretty early here and doing something like: sign in as admin > create a model > sign as a test user sounds like a big overhead for the test suite